external/avro/src/test/scala/org/apache/spark/sql/avro/AvroSuite.scala

```diff
@@ -2005,3 +2027,61 @@ class AvroV2Suite extends AvroSuite with ExplainSuiteHelper {
     }
   }
 }
+
+class AvroRowReaderSuite
```

Comment:
@bersprockets  @HyukjinKwon I just noticed recently that this suite is in `AvroSuite.scala`. Are there any specific reasons to not place it to `AvroRowReaderSuite.scala`?

Reply:
I don't think so. Feel free to move it there.

Reply:
Please, review the PR https://github.com/apache/spark/pull/32607